test/fixtures/echo-close-check.js

```diff
@@ -26,9 +26,15 @@ const fs = require('fs');
 
 process.stdout.write('hello world\r\n');
 
-var stdin = process.openStdin();
+const stdin = process.openStdin();
+
+let current;
+common.hijackStdout(function(data) {
```

Comment:
As there is an assertion in the listener function, it is better to have it wrapped in `common.mustCall`